default.nix

```diff
@@ -18,6 +18,7 @@
 , withLlvm  ? false
 , withDocs  ? true
 , mkFile    ? null
+, cores     ? 4
```

Comment:
not sure what a safe default would be but let's wait for someone to be angry at us for this to change it =)